components/organisms/ContinueRegistrationModalContent.tsx

```diff
@@ -0,0 +1,152 @@
+import { RegistrationStep } from "@/lib/name-registration/constants";
+import { LocalRegistrationNameData } from "@/lib/name-registration/types";
+import { useNameRegistration } from "@/lib/name-registration/useNameRegistration";
+import { BasicInfoKey, SocialAccountsKeys } from "../02-molecules";
+import { endNameRegistrationPreviouslyOpen } from "@/lib/name-registration/localStorage";
+import { useAccount } from "wagmi";
+import { Dispatch, SetStateAction } from "react";
+import { Button } from "@ensdomains/thorin";
+
+export const ContinueRegistrationModalContent = ({
+  name,
+  localNameRegistrationData,
+  setIsModalContinueRegistrationOpen,
+}: {
+  name: string;
+  localNameRegistrationData: LocalRegistrationNameData;
+  setIsModalContinueRegistrationOpen: Dispatch<SetStateAction<boolean>>;
+}) => {
+  const {
+    setRegistrationYears,
+    setCurrentRegistrationStep,
+    setAsPrimaryName,
+    setEnsResolver,
+    setTextRecords,
+    setCommitSubmitTimestamp,
+    setDomainAddresses,
+    nameRegistrationData,
+  } = useNameRegistration();
+
+  const { address } = useAccount();
+
+  const continueRegistration = () => {
+    //Set State of Registration Years
+    if (
+      localNameRegistrationData.registrationYears !== undefined &&
+      localNameRegistrationData.registrationYears >= 1
+    ) {
+      setRegistrationYears(localNameRegistrationData.registrationYears);
+    } else {
+      setCurrentRegistrationStep(RegistrationStep.RegistrationYears);
+      return;
+    }
+
+    //Set State of As Primary Name
+    if (localNameRegistrationData.asPrimaryName !== undefined) {
+      setAsPrimaryName(localNameRegistrationData.asPrimaryName as boolean);
+    } else {
+      setCurrentRegistrationStep(RegistrationStep.PrimaryName);
+      return;
+    }
+
+    //Set State of ENS Resolver
+    if (localNameRegistrationData.ensResolver !== undefined) {
+      setEnsResolver(localNameRegistrationData.ensResolver);
+    } else {
+      setCurrentRegistrationStep(RegistrationStep.ENSResolver);
+      return;
+    }
+
+    //Verifying if the commit timestamp was already saved
```

Comment:
Please put a space between // and the comment

```suggestion
    // Verifying if the commit timestamp was already saved
```